Jenkinsfile

```diff
@@ -9,20 +9,34 @@ pipeline {
         }
     }
 
+    tools {nodejs "Node 6.10.3"}
+
+    environment {
+        RESPONSE_OPERATIONS_UI_URL = 'http://response-operations-ui-python-ci.apps.devtest.onsclofo.uk/'
```

Comment:
Why do we set this here when the rest of the urls are configured using environment variables?